### ISSUE TITLE
Update Webmaker logo on other Moz sites etc

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-launch#38]
step 1 - gather a list of destinations

**Comment by edrushka** [comment]
@kevzawacki, do you have a list of channels that we could check to see where the logo is present?

**Comment by edrushka** [comment]
@xmatthewx we should assign this to someone with channel access. Maybe Kevin? Any ideas @HPaulJohnson?

**Comment by HPaulJohnson** [comment]
No list like this exists and an audit will need to be done to create one. We'll then need some resources to go through and make this update.

**Comment by edrushka** [comment]
Agree. This is probably quite a significant task. @xmatthewx do you want to file a ticket in /plan so we can resource it properly?

**Comment by HPaulJohnson** [comment]
Started a spreadsheet here that anyone can/should add to as they come across the logo. I likely won't have time to dedicate to this until next week: 

https://docs.google.com/a/mozilla.com/spreadsheets/d/19v3SW-oX68t4R8uZ9NMkRS36beyKAXIzHS0d3zgU26I/edit?usp=sharing

**Comment by KevZawacki** [comment]
@HPaulJohnson Thanks for the spreadsheet. I can start auditing next week, too.

**Comment by KevZawacki** [comment]
Spreadsheet updated with additional sites using the old logo:

https://docs.google.com/spreadsheets/d/19v3SW-oX68t4R8uZ9NMkRS36beyKAXIzHS0d3zgU26I/edit#gid=0

**Comment by xmatthewx** [comment]
nice. step 2 - swap logos starting with p1s.

**Comment by xmatthewx** [comment]
Filing bugs for the P1s in the spreadsheet. 
- [x] twitter
- [x] blog: https://github.com/mozilla/blog.webmaker.org/issues/15
- [x] teach: https://github.com/mozilla/teach.webmaker.org/issues/1073
- [ ] assets.mozilla.org & style guide: https://github.com/mozilla/webmaker-launch/issues/84
- [x] mozfest: https://github.com/mozilla/2015.mozillafestival.org/issues/141
- [x] Mozilla.org
- [x] transifex: https://github.com/mozilla/webmaker-android/issues/2285

**Comment by KevZawacki** [comment]
More bugs:
- [x] Google+: https://github.com/mozilla/learning-networks/issues/199
- [ ] Wikipedia: https://github.com/mozilla/webmaker.org/issues/1421
- [x] Mozilla.org: https://bugzilla.mozilla.org/show_bug.cgi?id=1172771

Note: SUMO is going to require a major overhaul. Logos, copy, updating/removing legacy tools, etc. https://support.mozilla.org/en-US/products/webmaker

**Comment by mmmavis** [comment]
Do we plan to update favicon as well?
e.g., http://mozilla.github.io/learning-networks/clubs/ and https://blog.webmaker.org/ (these are just the 2 examples I can think of)

**Comment by xmatthewx** [comment]
Good catch @mmmavis. @vazquez made a favicon for beta dot webmaker, but I'm not sure if we have optimized versions for different standards like iOS yet.

**Comment by xmatthewx** [comment]
@KevZawacki - besides mozilla assets and style guide, do you see any critical items remaining on this list?

**Comment by KevZawacki** [comment]
@xmatthewx SUMO is the only major item outstanding. We need to come up with a plan -- I'll open a ticket and drive the project.

**Comment by xmatthewx** [comment]
Right. Let's file than in the planning repo so you can get support across teams. Thanks for steering that.

**Comment by thisandagain** [comment]
- [ ] Firefox Tiles re: https://github.com/MozillaFoundation/plan/issues/541

**Comment by HPaulJohnson** [comment]
@LauraReynal thanks for flagging tile. We have provided an updated Webmaker tile and we're under the impression that it went live this week. 

You hadn't previously "pinned" that tile had you? In any case, I will folllow up with that team to make sure the new tile is being used.
